src/main/java/org/spongepowered/api/command/args/CommandElement.java

```diff
@@ -131,7 +71,81 @@ public void parse(CommandSource source, CommandArgs args, CommandContext context
      * @param src The source requesting usage
      * @return The formatted usage
      */
-    public Text getUsage(CommandSource src) {
-        return getKey() == null ? Text.of() : Text.of("<", getKey(), ">");
+    public abstract Text getUsage(CommandSource src);
+
+    public static abstract class Value<T> extends CommandElement {
+        private final Text key;
```

Comment:
This is either missing a `Nullable` annotation, or the constructor is missing a null check.